srcpkgs/audacious/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'audacious'
 #Keep in sync with audacious-plugins!
 pkgname=audacious
-version=4.0.5
+version=4.1
 revision=1
 build_style=gnu-configure
-configure_args="$(vopt_enable gtk) $(vopt_enable qt)"
+configure_args="$(vopt_enable gtk) $(vopt_enable qt) --enable-libarchive"
 hostmakedepends="pkg-config gettext glib-devel $(vopt_if qt qt5-host-tools)"
-makedepends="libSM-devel dbus-glib-devel libguess-devel $(vopt_if gtk gtk+-devel)
- $(vopt_if qt qt5-devel)"
+makedepends="libSM-devel dbus-glib-devel libguess-devel libarchive-devel libopenmpt-devel
```

Comment:
libopenmpt-devel should go in the makedepends for audacious-plugins, not audacious.

Reply:
Yes ofc... :)